### PR TITLE
fyne_settings: Do not allow scale 0 that causes infinite text size

### PR DESCRIPTION
**Pull request by fpabl0**
### Description:

Do not allow scale 0 that causes infinite text size.

### CODE REVIEW

**Comment by andydotxyz** [comment]
Nice catch, but I wonder - would it be better to put the check earlier so that it shows the default text size selected

**Comment by fpabl0** [comment]
> Nice catch, but I wonder - would it be better to put the check earlier so that it shows the default text size selected

Oh I see what you mean, you are right, will update :)

**Comment by fpabl0** [comment]
@andydotxyz updated :)

File: cmd/fyne_settings/settings/appearance.go [hunk 1]
```diff
@@ -128,6 +129,8 @@ func (s *Settings) createPreview() image.Image {
 
 	c.SetContent(tabs)
 	c.Resize(fyne.NewSize(380, 380))
+	// wait for indicator animation
+	time.Sleep(150 * time.Millisecond)
```
Comment (andydotxyz) [inline cmd/fyne_settings/settings/appearance.go:133]: I think it would be better if this referenced the constant instead - `canvas.DurationShort`

**Review by andydotxyz — CHANGES_REQUESTED** [review]

File: cmd/fyne_settings/settings/appearance.go (hunk 1 above)
Comment (fpabl0) [reply cmd/fyne_settings/settings/appearance.go]: You are right, updated! :)

**Review by andydotxyz — APPROVED** [review]

**Pull request merged**